openapi.json

```diff
@@ -356,6 +356,224 @@
+                             "minLength": 36,
+                             "maxLength": 36,
+                             "format": "uuid"
+                          },
+                          "meta": {
+                            "type": "object",
+                            "properties": {
+                              "count": {
+                                "type": "integer",
+                                "description": "Number of rules that were hit by the cluster. -1 is returned when no rules are defined for the cluster.",
+                                "example": "1"
+                              },
+                              "last_checked_at": {
+                                "type": "string",
+                                "format": "date",
+                                "example": "2020-01-23T16:15:59.478901889Z"
+                              }
+                            }
+                          },
+                          "report": {
+                            "type": "array",
+                            "items": {
+                              "type": "object",
+                              "properties": {
+                                "component": {
+                                  "type": "string",
+                                  "description": "The rule identifier for the hit rule.",
+                                  "example": "some.python.module"
+                                },
+                                "key": {
+                                  "type": "string",
+                                  "description": "The erroy key triggered for this rule in the cluster.",
+                                  "example": "SOME_ERROR_KEY"
+                                }
+                              }
+                            }
+                          }
+                        }
+                      }
+                    },
+                    "status": {
+                      "type": "string",
+                      "example": "ok"
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        "tags": [
+          "prod"
+        ]
+      }
+    },
+    "/clusters/reports": {
+      "post": {
+        "summary": "Returns the latest reports for the given list of clusters. Information about the org_id and user_id is taken from the token.",
+        "operationId": "getReportsForClustersPost",
+        "description": "Reports that are going to be returned are specified by list of cluster IDs that is part of request body.",
+        "parameters": [],
+        "requestBody": {
+            "description": "List of cluster IDs. Each ID must conform to UUID format. An example: `34c3ecc5-624a-49a5-bab8-4fdc5e51a266.",
+            "required": true,
+            "content": {
+              "text/plain": {
+                "schema": {
+                }
+            }
+          }
```

Comment:
I'm missing the parameter in the `"parameters"` or in the `"requestBody"`

Reply:
which parameter?

Reply:
I see now that the list of clusters should be in the body instead of parameters, but I miss the schema for the body content :-)